docs/03-developer-tools/00-sugar/05-reference/01-commands.md

```diff
@@ -155,6 +180,46 @@ sugar sign
 sugar sign -k creator.json
 ```
 
+## thaw
+
+This command thaws a single frozen NFT, or all frozen NFTs from a candy machine, if the candy machine's freeze settings allow it.
+
+Thaw a single NFT:
+
+```bash
+sugar thaw --candy-machine-id <candy_machine_id> <mint_account>
+```
+
+Thaw all NFTs from a candy machine:
+
+```bash
+sugar thaw --all --candy-machine-id <candy_machine_id>
+```
+
+In both commands, the candy machine id is optional and if left off Sugar will default to the id it finds in the cache file.
```

Comment:
In both commands, the candy machine id is optional, and if left  /*Added comma before and*/

Reply:
[Fixed.](https://github.com/metaplex-foundation/docs/pull/340/commits/e937a8967d014f97d810ff086455ccbbef39e55b)